src/hipscat/io/file_io/file_io.py

```diff
@@ -215,7 +216,7 @@ def read_parquet_dataset(
         format="parquet",
         **kwargs,
     )
-    return dataset
+    return (dir_pointer, dataset)
```

Comment:
Can we add something to the documentation explaining what this function returns? It seems a bit confusing to me that it returns an updated version of the pointer as well as the dataset.